css/cssom-view/scrollIntoView-vertical-rl-writing-mode.html

```diff
@@ -55,7 +55,12 @@
 <script>
 // In vertical-rl mode, X corresponds to the block axis and is oriented
 // leftward. Y corresponds to the inline axis and is oriented downward.
+// So the beginning edges are the top and right edges and the ending
+// edges are the bottom and left edges.
+
 // This assumes that the horizontal scrollbar is on the bottom side.
+// And it is compatible to the left side or right side vertical scroll bar.
```

Comment:
I wouldn't add the compatibility. If we want a comment, we should explain the issue with webkit below.

Anyway, I think we should remove the hacks for webkit and chromium and update the expectation files in their repos.